openshift-knative/eventing-kafka-broker
# Change default trigger consumergroup name and provide override (#3033)

* Change default consumergroup name to `knative-trigger-{{ .Namespace }}-{{ .Name }}` and provide override

* Add consumergroup override in trigger/v2

* Fix tests

* Fix triggerv2 tests

* Add triggerv2 test for reusing old consumergroup

* Add copyright notice

* Get rid of unused err

* Update `getKafkaTriggerConsumerGroup` to use new group id annotation

* Add kafka feature flag to namespaced trigger
